src/labs/ViewLab.tsx

```diff
@@ -129,7 +129,7 @@ const ViewLab = () => {
 
     if (permissions.includes(Permissions.CompleteLab)) {
       buttons.push(
-        <Button className="mr-2" onClick={onComplete} color="primary" key="labs.requests.complete">
+        <Button className="mr-2" color="primary" onClick={onComplete} key="labs.requests.complete">
```

Comment:
Any particular reason for switching order here?

Reply:
not particularly, just to mtch the order of the buttons.push a few lines above